webapp/cas/cmd/cas-stress/main.go

```diff
@@ -34,18 +34,20 @@ func main() {
 			return err
 		}
 
-		driver := agouti.ChromeDriver()
+		driver := agouti.ChromeDriver(agouti.ChromeOptions("args", []string{"headless"}))
 		if err := driver.Start(); err != nil {
 			return err
 		}
 
 		clickers := make([]func(*agouti.Page) error, 0)
 		url := c.String("with-matrix-url")
 		if url != "" {
+			url := url
```

Comment:
I think `golint` (or tslint) would choke on this as you're shadowing an existing variable.

Why do you need this in the first place?

You could do something like

```
if url := c.String("..."); url != ""{
}
```

Reply:
Ho, right, I have a bad habit of forgetting this go construct